packages/utils/odsp-utils/src/odspErrorUtils.ts

```diff
@@ -0,0 +1,235 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AxiosResponse } from "axios";
+
+export const offlineFetchFailureStatusCode: number = 709;
```

Comment:
This is all duplicated code, right? Can you please dedup it? 
I was wrong that odsp-driver uses odsp-utils, but I think nothing prevents from establishing this link and driver consuming all this code (minus axios specifics).

Unrelated to your changes - but do you know why we use axios? I.e. why exactly same code using fetch() is not used here? Is it' just historical thing, or there are some good reasons?

Reply:
I deduped the code. Odsp drive now uses utils code.
Renamed the package from odsp-utils to odsp-doclib-utils.

For axios, I will do that in a separate PR. This change is already big.

Reply:
Does the package really need to be renamed?  It seems unnecessary and the current naming doesn't seem "wrong" now that you're consuming it as expected

Reply:
Actually we(@curtisman and me) wanted to rename it to make sure that it is not a general dumping ground for odsp utils which exists within the odsp driver. So we renamed it to odsp-doclib-utils.